install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -0,0 +1,188 @@
+[[configuring-etcd-monitoring]]
+= Configuring etcd monitoring
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+If the `etcd` service does not run correctly, successful operation of the whole OpenShift cluster is in danger. Therefore, it is reasonable to configure monitoring of `etcd`.
```

Comment:
s/OpenShift/{product-title} 
Do this every time you have a generic reference to "OpenShift" in the change.